### ISSUE TITLE
Store Frame Node Id as new EditorState property

### ISSUE THREAD

**Issue by xQwexx** [prevwong/craft.js#201]
**Is your feature request related to a problem? Please describe.**
I am missing an reference from EditorState to the Frame node where the edited content start.
I just started to use this project :D so It possible that I couldn't find a better way feel free to give suggestions.

I found a workaround for it in the landing example:
With querySelector('.craftjs-renderer'), which is feels like a hack for this problem.

**Comment by matdru** [comment]
I'm not really sure what's the use case for that, `<Frame />` is responsible for simply loading initial state and acts as interactive region. That being said i think you can wrap Frame with another component or just a simple <div> and get a ref to that wrapper instead?

**Comment by xQwexx** [comment]
Yes, I just looking for Node where the edited content start and I think logically this information should be in the EditorState properties.
